src/com/hms_networks/americas/sc/logging/SocketLogger.java

```diff
@@ -77,8 +78,17 @@ private static void OPEN() {
       try {
         socketServerConnection =
             (ServerSocketConnection) Connector.open("socket://:" + SOCKET_PORT);
-      } catch (Exception e) {
-        Logger.LOG_DEBUG("SocketLogger connection could not open.");
+      } catch (ConnectionNotFoundException e) {
+        Logger.LOG_DEBUG("SocketLogger connection could not open (ConnectionNotFoundException).");

```

Comment:
Is there a reason for not logging the exception as well?

Reply:
I didnt want to eat up the queue space while its disconnected. The flexy seems to handle its sockets a bit strange so if you disconnect it often takes a few tries to restart the socketserver.